homework.txt

```diff
@@ -0,0 +1,48 @@
+
```

Comment:
Unnecessary to push

Reply:
Can you explain why I didn't need to git push?
I'm sure of why.

Also was the assignment done correctly?